azerothcore/azerothcore-wotlk
# fix(Core/Gameobject): Towercanno error location in Wintergrasp

## Changes Proposed:
-  adjusted wintergrasp "Flamewatch Tower" towercanno position

## Issues Addressed:

-  fix towercanno error location

## SOURCE:


## Tests Performed:

- It builds without errors
- Tested in-game

- Server:
AzerothCore rev. 893a9beeacc4+
Debian 11 x64
MySQL 8.0.33

- Client:
3.3.5a.12340 enUS
Windows 10 x64 LTSC

## How to Test the Changes:


- check towercanno error location: 
.go xyz 4448.713379 1955.148682 441.995178 571
-  confirm adjusted towercanno location: 
.go xyz 4489.819824 1955.636108 441.996002 571
